src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -37,6 +37,33 @@ public void BuildBasicApplicationRelease ()
 			}
 		}
 
+		[Test]
+		public void BuildBasicApplicationReleaseProfiledAot ()
```

Comment:
@radekdoulik I just realised these tests need a device right? 
So they are in the wrong project. The should be in the `MSBuildDeviceIntegration` project in `tests/MSBuildDeviceIntegration` where we have all the device based tests. 

It should also make use of the `HasDevices` property and `Ignore` the test if we don't have any devices attached.
For example https://github.com/xamarin/xamarin-android/blob/master/tests/MSBuildDeviceIntegration/Tests/InstallTests.cs#L233